### PR TITLE
Refs #32259 --  Modernized request attribute names. 

### PR DESCRIPTION
**Pull request by adamchainz**
ticket-32259

### CODE REVIEW

File: django/core/handlers/asgi.py [hunk 1]
```diff
@@ -85,42 +85,42 @@ def __init__(self, scope, body_file):
...
+            self.meta[corrected_name] = value
         # Pull out request encoding, if provided.
-        self._set_content_type_params(self.META)
+        self._set_content_type_params(self.meta)
         # Directly assign the body file to be our stream.
         self._stream = body_file
         # Other bits.
         self.resolver_match = None
 
-    @cached_property
```
Comment (auvipy) [inline django/core/handlers/asgi.py:98]: shouldn't this go through a deprecation period?

File: django/http/request.py [hunk 2]
```diff
@@ -402,6 +404,48 @@ def __iter__(self):
     def readlines(self):
         return list(self)
 
+    # Alias properties
+
```
Comment (auvipy) [inline django/http/request.py:408]: oh, you used aliasing for the old properties! didn't notice that first

**Comment by adamchainz** [comment]
Rebased for Django 4.0 and did another search for old attribute names.

**Comment by adamchainz** [comment]
I've agian rebased and searched through all docs/code for the old attribute names.

**Comment by felixxm** [comment]
Closing per ticket.

**Pull request closed without merging**